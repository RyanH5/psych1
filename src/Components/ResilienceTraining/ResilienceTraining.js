import React from 'react';

export const ResilienceTraining = () => {
  return (
    <div>
      <h1>ResilienceTraining</h1>
      <p className="first paragraph">
      I provide resiliency skills training to students, school staff, health care professionals and families. Resiliency training is an integrative skills-based model to help people develop self-awareness and skills to enhance physical, mental, and emotional health and well-being. This program integrates the best of movement, nutrition, and the psychology and practice of mindfulness to help restore resilience. Practicing resiliency skills can help calm your mind, deal skillfully with emotions, and open your heart to connection with self and others. These skills can help you function better with daily challenges and experience a more joyful life. Resiliency skills training has been extremely helpful to individuals, schools, families and businesses in developing core skills in restoring and strengthening mind, body and heart balance.
      </p>
    </div>
  )
}