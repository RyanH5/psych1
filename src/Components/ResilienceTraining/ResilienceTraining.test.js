import React from 'react';
import { shallow } from 'enzyme';
import { ResilienceTraining } from './ResilienceTraining';

describe('ResilienceTraining', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<ResilienceTraining />)

    expect(wrapper).toMatchSnapshot();
  });
});