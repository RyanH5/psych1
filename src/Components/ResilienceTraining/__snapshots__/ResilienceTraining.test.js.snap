// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResilienceTraining should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          ResilienceTraining
</h1>,
        <p
          className="first paragraph"
>
          I provide resiliency skills training to students, school staff, health care professionals and families. Resiliency training is an integrative skills-based model to help people develop self-awareness and skills to enhance physical, mental, and emotional health and well-being. This program integrates the best of movement, nutrition, and the psychology and practice of mindfulness to help restore resilience. Practicing resiliency skills can help calm your mind, deal skillfully with emotions, and open your heart to connection with self and others. These skills can help you function better with daily challenges and experience a more joyful life. Resiliency skills training has been extremely helpful to individuals, schools, families and businesses in developing core skills in restoring and strengthening mind, body and heart balance.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "ResilienceTraining",
        },
        "ref": null,
        "rendered": "ResilienceTraining",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "I provide resiliency skills training to students, school staff, health care professionals and families. Resiliency training is an integrative skills-based model to help people develop self-awareness and skills to enhance physical, mental, and emotional health and well-being. This program integrates the best of movement, nutrition, and the psychology and practice of mindfulness to help restore resilience. Practicing resiliency skills can help calm your mind, deal skillfully with emotions, and open your heart to connection with self and others. These skills can help you function better with daily challenges and experience a more joyful life. Resiliency skills training has been extremely helpful to individuals, schools, families and businesses in developing core skills in restoring and strengthening mind, body and heart balance.",
          "className": "first paragraph",
        },
        "ref": null,
        "rendered": "I provide resiliency skills training to students, school staff, health care professionals and families. Resiliency training is an integrative skills-based model to help people develop self-awareness and skills to enhance physical, mental, and emotional health and well-being. This program integrates the best of movement, nutrition, and the psychology and practice of mindfulness to help restore resilience. Practicing resiliency skills can help calm your mind, deal skillfully with emotions, and open your heart to connection with self and others. These skills can help you function better with daily challenges and experience a more joyful life. Resiliency skills training has been extremely helpful to individuals, schools, families and businesses in developing core skills in restoring and strengthening mind, body and heart balance.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            ResilienceTraining
</h1>,
          <p
            className="first paragraph"
>
            I provide resiliency skills training to students, school staff, health care professionals and families. Resiliency training is an integrative skills-based model to help people develop self-awareness and skills to enhance physical, mental, and emotional health and well-being. This program integrates the best of movement, nutrition, and the psychology and practice of mindfulness to help restore resilience. Practicing resiliency skills can help calm your mind, deal skillfully with emotions, and open your heart to connection with self and others. These skills can help you function better with daily challenges and experience a more joyful life. Resiliency skills training has been extremely helpful to individuals, schools, families and businesses in developing core skills in restoring and strengthening mind, body and heart balance.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "ResilienceTraining",
          },
          "ref": null,
          "rendered": "ResilienceTraining",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "I provide resiliency skills training to students, school staff, health care professionals and families. Resiliency training is an integrative skills-based model to help people develop self-awareness and skills to enhance physical, mental, and emotional health and well-being. This program integrates the best of movement, nutrition, and the psychology and practice of mindfulness to help restore resilience. Practicing resiliency skills can help calm your mind, deal skillfully with emotions, and open your heart to connection with self and others. These skills can help you function better with daily challenges and experience a more joyful life. Resiliency skills training has been extremely helpful to individuals, schools, families and businesses in developing core skills in restoring and strengthening mind, body and heart balance.",
            "className": "first paragraph",
          },
          "ref": null,
          "rendered": "I provide resiliency skills training to students, school staff, health care professionals and families. Resiliency training is an integrative skills-based model to help people develop self-awareness and skills to enhance physical, mental, and emotional health and well-being. This program integrates the best of movement, nutrition, and the psychology and practice of mindfulness to help restore resilience. Practicing resiliency skills can help calm your mind, deal skillfully with emotions, and open your heart to connection with self and others. These skills can help you function better with daily challenges and experience a more joyful life. Resiliency skills training has been extremely helpful to individuals, schools, families and businesses in developing core skills in restoring and strengthening mind, body and heart balance.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
